app/models/manageiq/providers/embedded_ansible/automation_manager/job.rb

```diff
@@ -3,8 +3,91 @@ class ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Job < ManageIQ::P
 
   require_nested :Status
 
+  def self.run_playbook(options, userid = nil)
+    userid ||= 'system'
+    name = build_name(options[:name])
+    description = options[:description]
+    info = options[:config_info]
+    playbook = ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Playbook.find(info[:playbook_id])
+    tower = playbook.manager
+    info[:inventory] = create_and_return_inventory(tower, name, description) if info[:inventory].nil?
+    _tower, params = ServiceTemplateAnsiblePlaybook.build_parameter_list(name, description, info)
```

Comment:
`build_parameter_list` can most likely be moved into a mixin to be shared by `ServiceTemplateAnsiblePlaybook`